PowerShellTools/Intellisense/Utilities.cs

```diff
@@ -103,6 +103,21 @@ internal static bool IsCaretInCommentArea(ITextView textView)
         }
 
         /// <summary>
+        /// Determins if caretis in string area.

```

Comment:
There are two kinds of way to measure types of tokens in PowerShell, the lower level is Token which has four kinds of string type: StringLiteral = 12, StringExpandable = 13,    HereStringLiteral = 14, HereStringExpandable = 15. And another is more abstract which is PSToken, it only has one kind of string type: String. So I use string area instead of string literal here to include a more general string type.